WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderExpandableTagsView.kt

```diff
@@ -209,7 +207,7 @@ class ReaderExpandableTagsView @JvmOverloads constructor(
             override fun overflowChipText(resources: Resources, hiddenChipsCount: Int): String {
                 return String.format(
                     resources.getString(R.string.reader_expandable_tags_view_overflow_indicator_expand_title),
-                    hiddenChipsCount
+                    hiddenChipsCount.toString()
```

Comment:
Minor (🔍): Consider removing the `String.format(...)` altogether, just like you did further above:

```kotlin
                return resources.getString(
                    R.string.reader_expandable_tags_view_overflow_indicator_expand_title,
                    hiddenChipsCount.toString()
                )
```

PS: Same applies to the `reader_expandable_tags_view_overflow_indicator_expand_title_new` further below.

Reply:
Good catch, thank you! Addressed in 7e9ba6c7863e9282358471dc8674df974fbc01a8